config/file.v1/file.go

```diff
@@ -21,18 +21,20 @@ type File struct {
 
 type CLIProperties struct {
 	// Upload configuration.
-	APIKey         string `yaml:"api_key,omitempty"`
-	Server         string `yaml:"server,omitempty"`
-	Fetcher        string `yaml:"fetcher,omitempty"` // Defaults to custom
-	Project        string `yaml:"project,omitempty"`
-	Title          string `yaml:"title,omitempty"`
-	Revision       string `yaml:"revision,omitempty"`
-	Branch         string `yaml:"branch,omitempty"`           // Only used with custom fetcher
-	ProjectURL     string `yaml:"project_url,omitempty"`      // Only used with custom fetcher
-	JIRAProjectKey string `yaml:"jira_project_key,omitempty"` // Only used with custom fetcher
-	Link           string `yaml:"link,omitempty"`
-	Team           string `yaml:"team,omitempty"`
-	Policy         string `yaml:"policy,omitempty"`
+	APIKey              string `yaml:"api_key,omitempty"`
+	Server              string `yaml:"server,omitempty"`
+	Fetcher             string `yaml:"fetcher,omitempty"` // Defaults to custom
+	Project             string `yaml:"project,omitempty"`
+	Title               string `yaml:"title,omitempty"`
+	Revision            string `yaml:"revision,omitempty"`
+	Branch              string `yaml:"branch,omitempty"`           // Only used with custom fetcher
+	ProjectURL          string `yaml:"project_url,omitempty"`      // Only used with custom fetcher
+	JIRAProjectKey      string `yaml:"jira_project_key,omitempty"` // Only used with custom fetcher
+	Link                string `yaml:"link,omitempty"`
+	Team                string `yaml:"team,omitempty"`
+	Policy              string `yaml:"policy,omitempty"`
+	ReleaseGroup        string `yaml:"release_group,omitempty"`
+	ReleaseGroupVersion string `yaml:"release_group_version,omitempty"`
```

Comment:
[nit] Hmm, I remember deciding that we preferred the compact syntax `--release_group name@version`. I don't think I have a strong opinion either way. If this is done and works and is simpler, let's go for it.

Just for posterity's sake, was there a particular reason you decided against the compact syntax?

Reply:
Yea good question, this is a note I left in the PR
```
I considered using one flag and splitting on a special character such as --release-group <name>@<version> but I decided to implement two separate flags. The main reason is to avoid unintended issues with splitting the strings on the backend and disallowing a certain special character. The second reason is to reduce confusion for the user who needs to manually add the character. I am favoring enforcing explicit declaration over the expense of the command looking a bit messy.
```
I understand some of the issues with having two flags, but I think being explicit is better in the long run as long as we fail when both aren't supplied.